### ISSUE TITLE
java.lang.IllegalArgumentException: attribute of the given name and type is not present in the managed type, for name:serialVersionUID

### ISSUE THREAD

**Issue by vanessaschissato** [Impetus/kundera#613]
I'm getting this exception when iterating results:

java.lang.IllegalArgumentException: attribute of the given name and type is not present in the managed type, for name:serialVersionUID

Shouldn't Kundera skip this field validation?

**Comment by chhavigangwal** [comment]
Please share the entity definition.

Chhavi

**Comment by vanessaschissato** [comment]
``` java
@Entity
@Table(name = "replication")
public class Replication extends BaseEntity<ReplicationId> {

    private static final long serialVersionUID = -8055485531216607863L;

    @EmbeddedId
         private ReplicationId id;

    // --

    public Replication() {
        id = new ReplicationId();
    }

    // --

    public ReplicationId getId() {
        return id;
    }

    public void setId(ReplicationId id) {
        this.id = id;
    }
}
```

``` java
@Embeddable
public class ReplicationId implements Serializable
{
        /************************

       DOESNT WORK IF I UNCOMMENT THIS SERIAL VERSION UID (OF THE EMBBEDED CLASS)

        private static final long serialVersionUID = -8055485531216607863L; 
        */

    @Column(name = "core")
    private String core;

    @Column(name = "date_hour")
    private String dateHour;

    @Column(name = "content")
    private String content;

    public String getCore() {
        return core;
    }

    public void setCore(String core) {
        this.core = core;
    }

    public String getDateHour() {
        return dateHour;
    }

    public void setDateHour(String dateHour) {
        this.dateHour = dateHour;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}
```

``` java
com.impetus.kundera.query.Query query = (com.impetus.kundera.query.Query) this.entityManagerProvider
                .get()
                .createQuery(
                        "SELECT entityName FROM " + this.getClass() + " entityName ");

query.setFetchSize(3);

return query.iterate();
```

**Comment by chhavigangwal** [comment]
Yes serialVersionUID should indeed be skipped for iterating over composite columns. Will be releasing the fix soon.
Would you like to share a pull request for the fix ?

Chhavi

**Comment by kkmishra** [comment]
Fixed in current trunk branch, please verify.

Thanks
Kuldeep

**Comment by chhavigangwal** [comment]
Fixed releasing with 2.13
